bciers/apps/registration/app/components/operations/registration/FacilityInformationPage.tsx

```diff
@@ -26,16 +26,11 @@ const FacilityInformationPage = ({
   step,
   steps,
 }: {
-  operation: UUID | "create";
+  operation: UUID;
   searchParams: FacilitiesSearchParams;
   step: number;
   steps: string[];
 }) => {
-  // Don't fetch operation if UUID is invalid or operation === "create"
```

Comment:
Removed because it's sounding like we're going to fetch in the higher component and pass the data down